2021-06-13 12:36:23,747 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-6b7c575cb6a42StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-6b7c575cb6a42StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,749 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: vm://n-6b7c575cb6a42StandardJMSBroker#4 failed: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#4) stopped.
2021-06-13 12:36:23,750 ERROR [org.apache.activemq.broker.TransportConnector] - Could not accept connection : java.lang.InterruptedException
2021-06-13 12:36:23,753 ERROR [n-6b7c575cb6a42.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,753 ERROR [n-6b7c575cb6a42.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-6b7c575cb6a42;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-6b7c575cb6a42}, Payload: null, Sender: msgbox:df@n-6b7c575cb6a42]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,755 WARN [n-6b7c575cb6a42.a-6b7c5a6019918.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c575cb6a42StandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,756 WARN [n-6b7c575cb6a42.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c575cb6a42StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c575cb6a42StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
nCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:05,142 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:05,153 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:05,177 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:06,144 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:06,164 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:06,184 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:07,152 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:07,173 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:07,192 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:08,156 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:08,181 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:08,193 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:09,162 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:09,184 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:09,192 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:10,169 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:10,192 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:10,199 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:11,171 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:11,195 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:11,205 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:12,177 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:12,198 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:12,214 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:13,179 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:13,205 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:13,217 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:14,182 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:14,215 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:14,225 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:15,183 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:15,221 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:15,232 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:16,192 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:16,224 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:16,237 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:17,201 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:17,232 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:17,243 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:18,210 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:18,233 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:18,243 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:19,217 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:19,244 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:19,244 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:20,222 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:20,246 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:20,248 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:21,227 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:21,249 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:21,253 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:22,230 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:22,254 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:22,254 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,233 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:23,261 ERROR [n-6b7c66dc68a02.a-6b7c69d5f0453.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,264 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:23,747 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.47:55144 failed: java.io.EOFException
2021-06-13 12:36:24,239 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:24,265 ERROR [n-6b7c66dc68a02.a-6b7c66c15e96a.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.aStarUpdate(WorkerBean.java:466)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:106)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,245 INFO [n-6b7c66dc68a02.a-6b7c636288c3c.BrokerBean] - running broker bean...
2021-06-13 12:36:25,571 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-6b7c66dc68a02StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-6b7c66dc68a02StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,577 ERROR [n-6b7c66dc68a02.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,577 ERROR [n-6b7c66dc68a02.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-6b7c66dc68a02;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-6b7c66dc68a02}, Payload: null, Sender: msgbox:df@n-6b7c66dc68a02]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,583 WARN [n-6b7c66dc68a02.a-6b7c636288c3c.CommunicationBean.jms] - could not close connection 
javax.jms.JMSException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#3) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.doSyncSendPacket(ActiveMQConnection.java:1445)
	at org.apache.activemq.ActiveMQConnection.close(ActiveMQConnection.java:687)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:110)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,584 WARN [n-6b7c66dc68a02.a-6b7c680d6b27c.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c66dc68a02StandardJMSBroker#5) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#5) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,585 WARN [n-6b7c66dc68a02.a-6b7c69d5f0453.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c66dc68a02StandardJMSBroker#7) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#7) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,585 WARN [n-6b7c66dc68a02.a-6b7c69d5f0453.CommunicationBean.jms] - could not close connection 
javax.jms.JMSException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#7) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:72)
	at org.apache.activemq.ActiveMQConnection.doSyncSendPacket(ActiveMQConnection.java:1450)
	at org.apache.activemq.ActiveMQConnection.close(ActiveMQConnection.java:687)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:110)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#7) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,586 WARN [n-6b7c66dc68a02.a-6b7c69133f0d7.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c66dc68a02StandardJMSBroker#9) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#9) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,586 WARN [n-6b7c66dc68a02.a-6b7c67600a63e.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c66dc68a02StandardJMSBroker#11) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#11) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,587 WARN [n-6b7c66dc68a02.a-6b7c66c15e96a.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c66dc68a02StandardJMSBroker#13) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#13) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:36:25,588 WARN [n-6b7c66dc68a02.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7c66dc68a02StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7c66dc68a02StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:37:57,268 ERROR [n-6b7e31eae5920.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-13 12:38:00,098 ERROR [n-6b7e3fa068b49.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-13 12:38:00,366 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - starting broker bean...
2021-06-13 12:38:00,972 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:01,976 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:01,986 INFO [n-6b7e3fa068b49.a-6b7e3f6b43030.WorkerBean] - WORKER RECEIVED de.dailab.jiactng.aot.gridworld.messages.ACOMessage@102a0f03
2021-06-13 12:38:01,998 INFO [n-6b7e3fa068b49.a-6b7e40bbdcf41.WorkerBean] - WORKER RECEIVED de.dailab.jiactng.aot.gridworld.messages.ACOMessage@1bed305f
2021-06-13 12:38:02,987 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:03,994 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:05,005 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:06,014 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:07,016 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:08,024 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:09,024 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:10,029 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:11,040 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:12,051 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:13,052 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:13,587 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.47:55159 failed: java.io.EOFException
2021-06-13 12:38:13,586 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-6b7e31eae5920StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-6b7e31eae5920StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:13,589 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: vm://n-6b7e31eae5920StandardJMSBroker#4 failed: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e31eae5920StandardJMSBroker#4) stopped.
2021-06-13 12:38:13,591 ERROR [org.apache.activemq.broker.TransportConnector] - Could not accept connection : java.lang.InterruptedException
2021-06-13 12:38:13,592 ERROR [n-6b7e31eae5920.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-6b7e31eae5920StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e31eae5920StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:13,593 ERROR [n-6b7e31eae5920.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-6b7e31eae5920;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-6b7e31eae5920}, Payload: null, Sender: msgbox:df@n-6b7e31eae5920]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-6b7e31eae5920StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e31eae5920StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:13,595 WARN [n-6b7e31eae5920.a-6b7e34e72ebf5.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e31eae5920StandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e31eae5920StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:13,595 WARN [n-6b7e31eae5920.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e31eae5920StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e31eae5920StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,061 INFO [n-6b7e3fa068b49.a-6b7e3c1e4f06e.BrokerBean] - running broker bean...
2021-06-13 12:38:14,708 ERROR [org.apache.activemq.broker.TransportConnector] - Could not accept connection : java.lang.InterruptedException
2021-06-13 12:38:14,708 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-6b7e3fa068b49StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-6b7e3fa068b49StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,709 ERROR [n-6b7e3fa068b49.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,711 ERROR [n-6b7e3fa068b49.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-6b7e3fa068b49;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-6b7e3fa068b49}, Payload: null, Sender: msgbox:df@n-6b7e3fa068b49]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,713 WARN [n-6b7e3fa068b49.a-6b7e3c1e4f06e.CommunicationBean.jms] - could not close connection 
javax.jms.JMSException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#3) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:72)
	at org.apache.activemq.ActiveMQConnection.doSyncSendPacket(ActiveMQConnection.java:1450)
	at org.apache.activemq.ActiveMQConnection.close(ActiveMQConnection.java:687)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:110)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,714 WARN [n-6b7e3fa068b49.a-6b7e40bbdcf41.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e3fa068b49StandardJMSBroker#5) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#5) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,716 WARN [n-6b7e3fa068b49.a-6b7e428128112.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e3fa068b49StandardJMSBroker#7) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#7) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,717 WARN [n-6b7e3fa068b49.a-6b7e41bde3061.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e3fa068b49StandardJMSBroker#9) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#9) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,718 WARN [n-6b7e3fa068b49.a-6b7e400aa2a95.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e3fa068b49StandardJMSBroker#11) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#11) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,720 WARN [n-6b7e3fa068b49.a-6b7e3f6b43030.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e3fa068b49StandardJMSBroker#13) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#13) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:38:14,721 WARN [n-6b7e3fa068b49.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-6b7e3fa068b49StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-6b7e3fa068b49StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:27,336 ERROR [n-299074992fe9.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-13 12:51:27,882 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - starting broker bean...
2021-06-13 12:51:27,986 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:27,987 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:28,994 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:28,995 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:29,995 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:29,996 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:30,705 ERROR [n-29913e58ccf3.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-13 12:51:31,004 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:31,005 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:32,004 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:32,005 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:33,005 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:33,006 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:34,008 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:34,009 WARN [n-299074992fe9.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-13 12:51:35,012 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:36,015 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:36,035 INFO [n-299074992fe9.a-299057ec5374.WorkerBean] - WORKER RECEIVED de.dailab.jiactng.aot.gridworld.messages.ACOMessage@348175a2
2021-06-13 12:51:36,049 INFO [n-299074992fe9.a-29907cf88e4b.WorkerBean] - WORKER RECEIVED de.dailab.jiactng.aot.gridworld.messages.ACOMessage@1d51de3b
2021-06-13 12:51:37,018 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:38,023 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:39,027 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:40,028 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:41,036 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:42,040 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:43,040 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:44,046 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:45,052 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:46,055 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:47,066 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:48,076 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:49,076 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:50,084 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:51,085 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:52,092 INFO [n-299074992fe9.a-299095ba9085.BrokerBean] - running broker bean...
2021-06-13 12:51:52,436 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.104:54818 failed: java.io.EOFException
2021-06-13 12:51:52,436 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-299074992fe9StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-299074992fe9StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,440 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: vm://n-299074992fe9StandardJMSBroker#14 failed: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#14) stopped.
2021-06-13 12:51:52,442 ERROR [org.apache.activemq.broker.TransportConnector] - Could not accept connection : java.lang.InterruptedException
2021-06-13 12:51:52,446 ERROR [n-299074992fe9.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-299074992fe9StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,447 ERROR [n-299074992fe9.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-299074992fe9;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-299074992fe9}, Payload: null, Sender: msgbox:df@n-299074992fe9]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-299074992fe9StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,448 WARN [n-299074992fe9.a-299095ba9085.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,450 WARN [n-299074992fe9.a-299098c8e886.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#5) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#5) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,451 WARN [n-299074992fe9.a-299057ec5374.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#7) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#7) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,452 WARN [n-299074992fe9.a-2990890cf99f.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#9) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#9) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,453 WARN [n-299074992fe9.a-29907cf88e4b.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#11) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#11) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,455 WARN [n-299074992fe9.a-299061dbe3af.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#13) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#13) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:52,456 WARN [n-299074992fe9.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-299074992fe9StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-299074992fe9StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:54,074 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-29913e58ccf3StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-29913e58ccf3StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:54,077 ERROR [n-29913e58ccf3.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-29913e58ccf3StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-29913e58ccf3StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:54,078 ERROR [n-29913e58ccf3.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-29913e58ccf3;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-29913e58ccf3}, Payload: null, Sender: msgbox:df@n-29913e58ccf3]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-29913e58ccf3StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-29913e58ccf3StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:54,080 WARN [n-29913e58ccf3.a-299135aa8de2.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-29913e58ccf3StandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-29913e58ccf3StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-13 12:51:54,082 WARN [n-29913e58ccf3.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-29913e58ccf3StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-29913e58ccf3StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
